core/include/kernel/ts_manager.h

```diff
@@ -13,6 +13,10 @@
 
 struct ts_ctx {
 	TEE_UUID uuid;
+#ifdef CFG_TA_PAUTH_ENABLE
+	uint64_t pauth_key_hi;
```

Comment:
This should perhaps be `uint64_t pauth_key[2];` instead to allow a single call to `crypto_rng_read()` in `tee_ta_init_user_ta_session()`